### PR TITLE
pails redo

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/usaco-615.mdx [hunk 1]
```diff
@@ -47,26 +21,91 @@ with open('pails.out', 'w') as fout:
...
+		int order = Integer.parseInt(input.nextToken());
+
+		int closest = 0;
+		// Try all possible combinations of the first & second bucket.
+		for (int first = 0; first <= order; first++) {
+			if (buck1 * first > order) {
+				break;
+			}
+			for (int second = 0; second <= order; second++) {
+				int n = buck1 * first + buck2 * second;
```
Comment (ryanchou-dev) [inline solutions/usaco-615.mdx:74]:
```suggestion
				int n = (buck1 * first) + (buck2 * second);
```
parens for clarity

**Review by ryanchou-dev — APPROVED** [review]
ur code is so impeccable im js nitpicking atp lol

**Review by ChuangSheep — APPROVED** [review]
🎉

**Comment by SansPapyrus683** [comment]
crap, didn't add parens to c++/python

**Pull request merged**